src/components/landing/ConnectingSection.svelte

```diff
@@ -1,5 +1,10 @@
 <script>
   import InfoSplitSection from '../InfoSplitSection.svelte'
+  import missionContent from '@contentful-entries/missionSection'
+  const content = missionContent.find(
+    mission => mission.key === 'connecting-section'
+  )
```

Comment:
`import content from '@contentful-entry/missionSection/connecting-section'`

Also add a line break here for consistency with [here](https://github.com/GTBitsOfGood/bog-web/pull/45/files#diff-1cbec1fd4337df0353d308927e4f451cR7).